File: algoritmos/06 - Funções/Lista01/respostas/questao07.py
def area_triangulo(base, altura):
    area = (base * altura) / 2
    return area

base = float(input("Digite a base: "))
altura = float(input("Digite a altura: "))

print(f"A área do triângulo é: {area_triangulo(base, altura):.0f} m2.")

File: estrutura de dados/3 - Funções/questoes/questao07.c
#include <stdio.h>

// Função que calcula a área de um triângulo
float calcular_area_triangulo(float base, float altura) {
    return (base * altura) / 2;
}

int main() {
    float base, altura;

    // Solicita ao usuário que insira a base e a altura do triângulo
    printf("Digite a base do triângulo: ");
    scanf("%f", &base);

    printf("Digite a altura do triângulo: ");
    scanf("%f", &altura);

    // Calcula e exibe a área
    printf("A área do triângulo é: %.2f\n", calcular_area_triangulo(base, altura));

    return 0;
}
